src/dataset-builder/DatasetBuilderDetails.ts

```diff
@@ -62,37 +62,36 @@ const TileDisplay = (props: DomainDisplayProps) => {
 };
 
 interface DatasetBuilderDetailsProps {
-  datasetId: string;
+  snapshotId: string;
 }
 
-export const DatasetBuilderDetails = ({ datasetId }: DatasetBuilderDetailsProps) => {
-  const [datasetDetails, loadDatasetDetails] = useLoadedData<DatasetModel>();
-  const [datasetRoles, loadDatasetRoles] = useLoadedData<string[]>();
+export const DatasetBuilderDetails = ({ snapshotId }: DatasetBuilderDetailsProps) => {
```

Comment:
Some renaming still needed here, although perhaps the naming could be `dataRelease` instead of `snapshot`?

Reply:
I'm worried it would be confusing to introduce another name for snapshots.